### ISSUE TITLE
The documentation is very poor on how to use React Tabs with Router and links

### ISSUE THREAD

**Issue by davidradl** [carbon-design-system/carbon#5882]
The documentation is very poor on how to use React Tabs with Router and links. I understand there used to be much more information in this area on the web site. I think this is a basic tabs use case.

I would have liked to have seen a working Carbon React tabs example that changed the url as the tabs changed. At the moment I am still unsure how to work with the React Tabs and links.
 
I notice that https://react.carbondesignsystem.com/?path=/story/tabs--default has no readme. So there are also no basic instruction on how to install this component.  

I notice that Link does not have a readme either. 

This seems similar to #1503 which is closed, but the docs still seem inadequate.

**Comment by emyarod** [comment]
does this example in our repository help?

https://codesandbox.io/s/github/carbon-design-system/carbon/tree/main/packages/react/examples/react-router

https://github.com/carbon-design-system/carbon/tree/main/packages/react/examples/react-router

**Comment by dakahn** [comment]
Going to close this question as answered.

**Comment by kristinochka** [comment]
New url that works: https://codesandbox.io/s/github/carbon-design-system/carbon/tree/main/packages/react/examples/react-router

**Comment by ninjasun** [comment]
I believe a tipic scenario is to use tabs as nested routes. 
Something like [https://codesandbox.io/s/stoic-flower-hp153?file=/src/App.js](url)

Is there a way to avoid generation of  tabpanel as it's not always a child of Tab.

thanks

**Comment by reinisg** [comment]
sorry, but all examples are not working..
